Apps/FlightSoftware/Watchdog/src/main.c

```diff
@@ -92,7 +94,8 @@ void enterMode(enum rover_state newstate) {
         stopChargingBatteries();
 
         __delay_cycles(12345678); //give fuel gauge [50 ms] & wifi [~750 ms] time to start up
-        initializeFuelGauge();
+//        initializeFuelGauge();
+        I2C_Sensors__initializeFuelGaugeBlocking();

```

Comment:
Do we still need to have the delay cycles if we are blocking to initialize the fuel gauge?

Reply:
I am not sure, although probably? Seeing as Matt's code is infinitely more robust than mine and should wait for a response from the fuel gauge before moving on. I can look into the code tomorrow to verify that assumption

Reply:
It's also needed for the wifi chip to start up (unless we find a better way than just delaying to ensure it is ready when we power on Hercules) so probably can't remove it anyway